LESSON2.md

```diff
@@ -361,8 +361,3 @@ stdin.on('end', () => {
 
 *This is exactly the same as our read stream example at the top of this readme, as
 process.openStdin() is a read stream!*
-
-### Project
-
-With what we have covered in these two lessons, we now have everything we need to build some command-line tools! Move onto
```

Comment:
@jsms90 @joeylouise Sorry to be a pain, I'm just unsure if this is the best approach for updating this resource in the long-term. If next cohort the mentors decide to not cover the project in the research afternoon, or use only one lesson of the workshop as Marko did, then we'd need to remember to update this line to reflect that.

I think whoever delivers a workshop should tell the students what parts of it they want them to cover. Happy to go with this if you both disagree though.